lib/init.js

```diff
@@ -176,7 +182,8 @@ module.exports = function(opts) {
   // Set interactive boolean off of CLI flags
   options = opts;
 
-  console.log('Initializing Tessel 2 Project...');
+  log.info('Initializing Tessel 2 Project...');
+  log.spinner.start();
```

Comment:
If this is started here, does `log.spinner.start()` need to be in all the async functions like `loadNpm`? Probably doesn't hurt to call it in each async function, just wondering.